lib/orbital/api.rb

```diff
@@ -2,6 +2,8 @@ module Killbill #:nodoc:
   module Orbital #:nodoc:
     class PaymentPlugin < ::Killbill::Plugin::ActiveMerchant::PaymentPlugin
 
+      FIVE_MINUTES_AGO = (1 * 300)
```

Comment:
Right, the data is in the database, but a `like` query on that table won't be scalable (full table scan) as we record more and more payments (unless we go to a warehouse, but ETLs are often delayed, so not useful during outages). That's why I'm looking for a different solution (log storage search engine, indexed by timestamp).

Ideally, we would only log when we overwrite the payment status the first time (typically in `authorizePayment`).